requirements.txt

```
@@ -3,4 +3,4 @@ element-lab==0.1.0b1
 element-animal==0.1.0b0
 element-session==0.1.0b0
 ipykernel
-pynwb==1.4.0
+pynwb==2.0.0
```

testing the full export package with 2.0 has been a back burner item for me. If this works across all element exports, great. If we don't know yet, I'd go with the following
```suggestion
pynwb>=1.4.0
```

---

This should work. The biggest issue here is the support for h5py 3.x that is enabled by pynwb 2.x. Without this, certain features of h5py will not be accessible, such as reading remote files on s3.

---

Thanks @CBroz1.  I tested with pynwb 2.0.0 and it exports without error.  Please also test and let me know if you run into any issues.

---

To follow-up on this, I pulled from your fork @kabilar and attempted the `docker-compose` command
`docker-compose -f docker-compose-test.yaml up --build`

but get the following error
`ERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?`
Could there be an issue with the dockerfile updates?

---

Ah yes, if using the `Dockerfile.test`, change the base image to `datajoint/djbase:py3.9-debian-fcd8909`.

---

I have now update the `Dockerfile.test`.

---

Hi @CBroz1, also moved Docker and Compose files to `docker` subdirectory.  `git mv` wasn't working for `docker-compose-dev.yaml` so original file is deleted and a new file is created.